.gitignore

```diff
@@ -1,19 +1,20 @@
 *.egg-info
 *.pyc
+__pycache__/
 .env
 .tox
 .TAGS
 build
 dist/
 MANIFEST
 _trial_temp
-.coverage
+/.coverage
+/.coverage.*
```

Comment:
Would `.coverage*` work?

Reply:
Though we don't have one in this repo, `.coverragerc` is a thing so I'd prefer to be specific.